integration/tests/cook/util.py

```diff
@@ -1635,16 +1640,17 @@ def _get_compute_cluster_factory_fn():
 
 
 def using_kubernetes():
-    return get_kubernetes_compute_cluster() is not None
-
+    return os.getenv("COOK_TEST_COMPUTE_CLUSTER_TYPE","mesos") == "kubernetes"
 
 def using_mesos():
-    return _get_compute_cluster_factory_fn() == 'cook.mesos.mesos-compute-cluster/factory-fn'
-
+    return os.getenv("COOK_TEST_COMPUTE_CLUSTER_TYPE","mesos") == "mesos"
 
 def has_one_agent():
-    return node_count() == 1
-
+    if using_kubernetes():
+        # Kubernetes always has more than one agent.
+        return False
```

Comment:
Why?

Reply:
Simplifies this code. I hate that we run these tests in so many environments and randomly disable subsets of tests. No production environment will have only a single node.